ddh-intro.md

```diff
@@ -2,31 +2,34 @@
 
 ![](http://docs.duckduckhack.com/assets/hack_search_engine.png)
 
-**We're a community dedicated to making DuckDuckGo the best search engine for programmers. We leverage 3rd party APIs, scrape external data sets, and write well-placed code to create what we call *Instant Answers*.**
+**We're a community dedicated to making DuckDuckGo the best search engine for programmers, and you can help!**
 
-You can think of [Instant Answers](https://duck.co/ia?topic=programming) as "perfect search results." They resolve queries above any ads or regular search results. Millions of searches on DuckDuckGo trigger Instant Answers, like [MDN Javascript](https://duckduckgo.com/?q=javascript+array&ia=about) and [Regex Cheat Sheet](https://duckduckgo.com/?q=regex+cheat+sheet&ia=cheatsheet&iax=1). 
 
-Our Instant Answers are created and maintained by a community of open source developers around the world. Just like the Wikipedia community creates articles for topics – the DuckDuckHack community creates Instant Answers for searches! 
+## Programmin Mission
 
-## Get Started
+The Programming Mission empowers the community to create Instant Answers for reference, libraries, help, and tools.
+Our goal is to have a high quality, relevant Instant Answer for every search.
```

Comment:
@russellholt Tweaked it, not exactly the way you proposed, I still like the separate Programming Mission header. How does it look?